extensions/proposals/WEBGL_element_texture/extension.xml

```diff
@@ -0,0 +1,298 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<proposal href="proposals/WEBGL_element_texture/">
+  <name>WEBGL_element_texture</name>
+
+  <contact> <a href="https://www.khronos.org/webgl/public-mailing-list/">WebGL
+  working group</a> (public_webgl 'at' khronos.org) </contact>
+
+  <contributors>
+    <contributor>Byungseon Shin, LG Electronics</contributor>
+
+    <contributor>Andrei Volykhin, LG Electronics</contributor>
+
+    <contributor>Mark Callow, HI Corporation</contributor>
```

Comment:
Please change to Edgewise Consulting. I left HI a couple of years ago.

Reply:
I used information from old extension, i will change it.